### PR TITLE
README documentation update

### LINKED ISSUE #51: README documentation update
### Describe the solution you'd like

We need to update the `README` in order to add the documentation about the main changes in version v0.2.0`.

### Describe alternatives you've considered

- [x] Add the documentation about the accounts endpoints added.
- [x] Add the documentation about the success response format.
- [x] Add the documentation about the error response format.

### PR DESCRIPTION
**Pull request by CristhianRodriguezMolina**
## Description

Fixes #51

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,154 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
...
+| `status`  | Integer | Status of the request.                                            |
+
+
+### Accounts
+
+Group of endpoint to manage the account resource.
+
+#### **Create account**
+    
+The `Create account` endpoint allows us to create accounts, this endpoint returns us the necessary to manage our account in Mintacoin, the address (Mintacoin public key), the signature (Mintacoin secret key), and the seed words to recover our signature.
```
Comment (miguelnietoa) [inline README.md:62]:
```suggestion
It creates an account on Mintacoin and the given blockchain.
This endpoint returns the necessary to manage an account in Mintacoin: **address** (Mintacoin public key), **signature** (Mintacoin secret key), and **seed words** (to recover the signature).
```

File: README.md [hunk 2]
```diff
@@ -20,6 +20,154 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
...
+| `address`    | String | This is the public key for your Mintacoin account.                                                                    |
+| `signature`  | String | This is the secret key for your Mintacoin account. keep it safe and don't share it.                                   |
+| `seed_words` | String | These are 12 words that you can use to recover your `signature` if you lost it, keep them safe, and don't share them. |
+
+---
+
+#### **Recover signature**
+    
+The `Recover signature` endpoint allows recovering our signature with the address and the seed word of our account.
+
```
Comment (miguelnietoa) [inline README.md:106]: @CristhianRodriguezMolina I consider `signature`, `address` and `seed_words` with code format and in a more direct description could look better. What do you think?
```suggestion
Recovers the `signature` with the `address` and the `seed_words` of an account.
```

File: README.md [hunk 3]
```diff
@@ -20,6 +20,154 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
...
+The `Create account` endpoint allows us to create accounts, this endpoint returns us the necessary to manage our account in Mintacoin, the address (Mintacoin public key), the signature (Mintacoin secret key), and the seed words to recover our signature.
+
+
+##### HTTP request
+
+```http
+POST /accounts
+```
+
+##### Curl request
```
Comment (miguelnietoa) [inline README.md:71]: @CristhianRodriguezMolina Taking a good look at it... I think we could remove "HTTP request" and change "Curl request" with "Example request" 🤔

**Review by miguelnietoa — CHANGES_REQUESTED** [review]
@CristhianRodriguezMolina good documentation! I've added some comments, let me know what you think 💯

File: README.md [hunk 4]
```diff
@@ -20,6 +20,154 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
...
+  "status": 200 | 201
+}
+```
+
+##### Response parameters
+
+| Parameter | Type    | Description                                                       |
+| --------- | ------- | ----------------------------------------------------------------- |
+| `data`    | Map     | This contains the data returned by the endpoint you're accessing. |
+| `status`  | Integer | Status of the request.                                            |
```
[the remark this answers is not in the document]
Comment (CristhianRodriguezMolina) [reply README.md:53]: You're right we can specify that it is an HTTP status code

File: README.md [hunk 5]
```diff
@@ -20,6 +20,154 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
...
+```
+
+##### Curl request
+
+```bash
+curl -X POST \
+     -H 'Content-Type: application/json' \
+     -d '{"blockchain": "stellar"}' \
+     "https://sandbox.api.mintacoin.co/accounts"
+```
```
[the remark this answers is not in the document]
Comment (CristhianRodriguezMolina) [reply README.md:78]: I'll add a section for the body params of each endpoint.

File: README.md (hunk 3 above)
Comment (CristhianRodriguezMolina) [reply README.md:71]: What do you think about something like this: 

image

It would look like this:

image

Comment (juanhurtado10) [reply README.md:71]: @CristhianRodriguezMolina @miguelnietoa why not use Gitbook to place this type of documentation?

Comment (juanhurtado10) [reply README.md:71]: my 5 cents: 
1. Point `docs.mintacoin.co` to Gitbook.
2. Synchronize the Gitbook space with the GitHub repo.
3. Write the documentation in .md which automatically syncs the Gitbook space.

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

**Review by miguelnietoa — APPROVED** [review]

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**